Makefile

```diff
@@ -20,6 +21,26 @@ endif
 	go test ./...
 	@stty sane
 
+warplark: check-warplark
+ifndef WARPLARK
+	@echo "Building and installing warplark..."
+	go install github.com/warptools/warplark
+endif
+
+update-warplark:
+	go install github.com/warptools/warplark@latest
+	sha256sum $$(which warplark) | cut -f1 -d' ' > tools/warplark.sha256
```

Comment:
this is dumb. the hash will change based on which go compiler you are using.
We would have to pin the compiler version as well.